src/util/regExpUtils.ts

```diff
@@ -1,3 +1,10 @@
+/**
+	* Fetches a RegExp's group's values by its number for all matches
+	* @param {string} textBasis - The text to be fetched from
```

Comment:
Why is the param here different to the one in the signature of the actual function?